articles/multifactor-authentication/api/email.md

```diff
@@ -0,0 +1,147 @@
+  - how-to
+useCase:
+  - customize-mfa
+---
+
+# Associate an Email Authenticator
+
+In this tutorial, you'll learn how to configure your application so users can use email authenticators.
+
+<%= include('./_authenticator-before-start') %>
+
+## 1. Enable email authenticators with the Management API
+
+Start by enabling email authenticators with the [Management API](/api/management). To do this, make a `PUT` request to the [/api/v2/guardian/factors/email](/api/management/v2/#!/Guardian/put_factors_by_name) endpoint. You'll need a [Management API Token](/api/management/v2/tokens) to perform the request:
+
+```har
+{
+  "method": "PUT",
+  "url": "https://${account.namespace}/api/v2/guardian/factors/email",
+  "headers": [
+    { "name": "Authorization", "value": "Bearer MANAGEMENT_API_TOKEN" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{\"enabled\":\"true\"}"
+  }
+}
+```
+
+## 2. Get the MFA Token
+
+When a user begins the authorization process without an active authenticator associated with their account, they will trigger the an `mfa_required` error when calling the `/oauth/token` endpoint. For example:
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/oauth/token",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{\"grant_type\":\"password\",\"username\": \"user@example.com\",\"password\": \"pwd\",\"audience\": \"https://someapi.com/api\", \"scope\": \"read:sample\", \"client_id\": \"${account.clientId}\", \"client_secret\": \"YOUR_CLIENT_SECRET\"}"
+  }
+}
+```
+
+The `mfa_required` error will look like this:
+
+```json
+{
+  "error": "mfa_required",
+  "error_description": "Multifactor authentication required",
+  "mfa_token": "Fe26...Ha"
+}
+```
+
+In the next step, you can use the `mfa_token` value to request association of a new authenticator.
+
+## 3. Request association of the authenticator
+
+Next, make a `POST` request to the `/mfa/associate` endpoint to request association of your authenticator. Remember to use the MFA token from the previous step.
+
+To associate an authenticator where the challenge type is an email containing a code the user provides, make the following `POST` request to the `/mfa/associate` endpoint. Be sure to replace the placeholder values in the example below as appropriate.
+
+```har
+{
+    "method": "POST",
+    "url": "https://${account.namespace}/mfa/associate",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer MFA_TOKEN"
+    }],
+    "postData": {
+        "mimeType": "application/json",
+        "text": "{ \"authenticator_types\": [\"oob\"], \"oob_channels\": [\"email\"]}"
+    }
+}
+```
+
+If successful, you'll receive a response like this:
+
+```json
+{
+  "authenticator_type": "oob",
+  "oob_channel": "email",
+  "recovery_codes": [ "N3BGPZZWJ85JLCNPZBDW6QXC" ],
+  "oob_code": "ata6daXAiOi..."
+}
+```
+
```

Comment:
Can we add a p describing what happens next? ie. The users receives an email on their inbox ... grabs the code and puts it back into the app ... enrollment is completed.

Reply:
We should also add a note describing how the email template can be customized.